### PR TITLE
tests: add tests for diregapic 

### PR DESCRIPTION
**Pull request by georgiyekkert**
This is the same as  PR #276. recreated to see if kokoro will pick up changes

### CODE REVIEW

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Integration tests
```
Comment (vam-google) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/readme.md:1]: Please rename this file to README.md (upper case) for consistency

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITSmokeInstancesTest extends BaseTest {
+  private static InstancesClient instancesClient;
+  private static ZoneOperationsClient operationsClient;
+  private static List<Instance> instances;
+  private static final String DEFAULT_IMAGE =
+      "https://www.googleapis.com/compute/v1/projects/debian-cloud/global/images/debian-7-wheezy-v20150710";
```
Comment (vam-google) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java:38]: Is it guaranteed that this image will not go away on its own (i.e. will not get deleted as part of some cleanup procedure independent from this test)?

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+  @AfterClass
+  public static void tearDown() {
+    zonesClient.close();
+  }
+
+  @Test
+  public void testList() {
+    ZonesClient.ListPagedResponse response = zonesClient.list(DEFAULT_PROJECT);
+    boolean presented = Boolean.FALSE;
```
Comment (vam-google) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:43]: Why not just?
```java
boolean presented = false;
```

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    for (Zone element : response.iterateAll()) {
+      if (element.getName().equals(DEFAULT_ZONE)) {
+        presented = Boolean.TRUE;
+      }
+    }
+    Assert.assertTrue(presented);
+  }
+
+  @Test
+  public void testPaginationMaxResults() {
```
Comment (vam-google) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:53]: Can you please add the same test  (with max results) with built-in pagination (the one which just does response.iterateAll())? To make sure that max results is appreciated.

**Review by vam-google — APPROVED** [review]
LGTM with few minor comments

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java (hunk 2 above)
Comment (georgiyekkert) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java]: Done

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java (hunk 4 above)
Comment (georgiyekkert) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:53]: Done

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java (hunk 2 above)
Comment (georgiyekkert) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java]: Changed to just debian-10, it shouldn't be deprecated in ~5 years

**Pull request closed without merging**